keps/prod-readiness/sig-network/2086.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2086
+alpha:
```

Comment:
PRR is already filled out in the KEP https://github.com/kubernetes/enhancements/blob/699fd3769fd3cd1f6846020675a330cdcee5f5af/keps/sig-network/2086-service-internal-traffic-policy/README.md#production-readiness-review-questionnaire but it needs a review